man/scrot.txt

```diff
@@ -182,9 +182,33 @@ formats where the quality and compression are tied together (e.g JPEG),
 compression will be ignored. And for image formats where quality and
 compression can be independently set (e.g WebP, JXL), both flags are respected.
 
+EXAMPLES
+
+Following are a couple examples of using scrot with other utilities.
+
+Copying screenshot to clipboard using xclip(1):
+
+  $ scrot - | xclip -selection clipboard -target image/png
+
+Taking a screenshot and optimizing it's size with optipng(1):
```

Comment:
The correct English here is "its". Possessive determiners don't have apostrophes:
>he/his
>she/her
>it/its